metabase/metabase
# [E2E] Upgrade cross-version Cypress configs to v10 format

### Status
PENDING CI && PENDING REVIEW

### What does this PR accomplish?
- This is most likely the very last step in the effort to upgrade Cypress to the latest version
- Configs changed and the cross-version tests need to reflect that. This PR takes care of it.

### How to test?
- All tests should still pass in CI
- Since we run these on a schedule, [I tested them in the current branch](https://github.com/metabase/metabase/pull/25711/commits/eadeb7e63dbbb6f098afaa07a892b4bd323a735d) and everything passed: https://github.com/metabase/metabase/actions/runs/3153646678
